bigbluebutton-html5/imports/api/breakouts/server/publishers.js

```diff
@@ -1,23 +1,27 @@
 import { Meteor } from 'meteor/meteor';
 import Breakouts from '/imports/api/breakouts';
+import Users from '/imports/api/users';
 import Logger from '/imports/startup/server/logger';
 
-function breakouts(credentials, moderator) {
+function breakouts(credentials, moderator = false) {
   const {
     meetingId,
     requesterUserId,
   } = credentials;
   Logger.info(`Publishing Breakouts for ${meetingId} ${requesterUserId}`);
 
   if (moderator) {
-    const presenterSelector = {
-      $or: [
-        { parentMeetingId: meetingId },
-        { breakoutId: meetingId },
-      ],
-    };
+    const User = Users.findOne({ userId: requesterUserId });
+    if (User.moderator) {
```

Comment:
I think we should add a check for User was found: `if (!!User && User.moderator) {...}` - in all three cases in this PR